examples/openssh.rs

```diff
@@ -0,0 +1,23 @@
+//! Expected `Cargo.toml` dependencies configuration:
+//!
+//! ```
+//! [dependencies]
+//! openssh-sftp-client = { version = "0.13.6", features = ["openssh"] }
+//! ```
+
+#[cfg(not(feature = "openssh"))]
+compile_error!("Please enable features openssh!");
```

Comment:
This requires users to enable the openssh feature for their own crate, not for `openssh-sftp-client`. I think it's better to remove it.

Reply:
Thanks, I've removed it.